Packs/GitLab/Integrations/GitLab/GitLab.py

```diff
@@ -0,0 +1,1381 @@
+                           'Assignee': response.get('assignee', ''),
+                           'CreatedBy': response.get('author', {}).get('name', '')}
+    if response.get('author'):
+        human_readable_dict['CreatedBy'] = response.get('author').get('name', '')
+    if response.get('milestone'):
+        human_readable_dict['Milstone'] = response.get('milestone').get('title', '')
+    human_readable = tableToMarkdown('Update Issue', human_readable_dict, removeNull=True, headers=headers)
+    return CommandResults(
+        outputs_prefix='GitLab.Issue',
+        outputs_key_field='Iid',
+        readable_output=human_readable,
+        outputs=response,
+        raw_response=response
+    )
+
+
+def version_get_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Gets the current version.
+    Args:
+        client (Client): Client to perform calls to GitLab services.
+        args (Dict[str, Any]): XSOAR arguments:
+            - No arguments needed.
+    Returns:
+        (CommandResults).
+        'GitLab \n version:
+    """
+    response = client.version_get_request()
+    version = response['version']
+    revision = response['revision']
+    human_readable_string = f'GitLab Version \n version: {version}\n reversion: {revision} '
+    command_results = CommandResults(
+        outputs_prefix='GitLab.Version',
+        readable_output=human_readable_string,
+        outputs_key_field='revision',
+        outputs=response,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def file_get_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Allows to receive information about file in repository like name, size, content..
+    Args:
+        client (Client): Client to perform calls to GitLab services.
+        args (Dict[str, Any]): XSOAR arguments:
+            - 'file_path' (Required): The file path.
+            - 'ref': The branch to retrieve the file from, default is main
+
+    Returns:
+        (CommandResults).
+    """
+    branch = args.get('ref', 'master')
+    file_path = args.get('file_path', '')
+    headers = ['FileName', 'FilePath', 'Ref', 'ContentSha', 'CommitId', 'LastCommitId', 'Size']
+    response = client.file_get_request(file_path, branch)
+    human_readable_dict = {'FileName': response.get('file_name', ''),
+                           'FilePath': response.get('file_path', ''),
+                           'Ref': response.get('ref'),
+                           'ContentSha': response.get('content_sha256', ''),
+                           'CommitId': response.get('commit_id', ''),
+                           'LastCommitId': response.get('last_commit_id', ''),
+                           'Size': response.get('size', '')}
+    human_readable = tableToMarkdown('Get File', human_readable_dict, removeNull=True, headers=headers)
+    return CommandResults(
+        outputs_prefix='GitLab.File',
+        outputs_key_field=['path', 'ref'],
```

Comment:
```suggestion
        outputs_key_field='path',
```
Make sure to fix this in all relevant places